create-fetchival.js

```diff
@@ -13,16 +13,20 @@ function createFetchival({ fetch }) {
     // Unlike fetchival, don't silently ignore and override
     if (opts.body) throw new Error('unexpected pre-set body option')
 
+    const headers = {
+      Accept: 'application/json',
+      'Content-Type': 'application/json',
+      ...opts.headers,
+    }
+    // If a consumer has set the non-json content type, we don't stringify the body.
+    const shouldStringify = headers['Content-Type'] === 'application/json'
```

Comment:
theoretically there could be others like application/vnd.api+json but in practice maybe we don't have them

Reply:
can we stringify for application/json and startsWith('application/') && endsWith('+json')? What does that miss?